### ISSUE TITLE
Changes in *BUCKLE definition

### ISSUE THREAD

**Issue by lucas-bueno777** [Dhondtguido/CalculiX#57]
I want to ask for two main things: the first is about the default accuracy in *Buckle cards. As the documentation states, the default accuracy is 0.01, and most of the time it should not be changed.

![image](https://github.com/Dhondtguido/CalculiX/assets/108995763/be386ec0-8a69-4d46-930e-df596880c9a7)

However, I did some tests that are presented in the [CalculiX forum](https://calculix.discourse.group/t/buckling-of-composite-panels/1624/34?u=lucas_bueno) and when the default settings are used, Pardiso, PastiX and Spooles fail to find the first eigenvalue when only one is requested. However, if we change the accuracy to 0.0001 (100 times more accurate than default), all solvers now can handle the search for the first eigenvalue properly with absolutely minimal changes in time spent in the run for my test case. Therefore, my first point is to improve the default accuracy of the *Buckle card from 0.01 to 0.0001 at least.

![image](https://github.com/Dhondtguido/CalculiX/assets/108995763/7767bd96-9489-415d-a476-c3f6a32da336)

The second thing I would like to ask is to add control parameters to change the buckling definition the same way they are presented in the *Frequency card. This way, one can easily avoid negative eigenvalues and search only within a defined range if is desired.

![image](https://github.com/Dhondtguido/CalculiX/assets/108995763/bd35d5ef-a8a6-430a-8c4d-da4b3dc3ad38)

Best regards,
Lucas

**Comment by victorkemp** [comment]
I agree about the accuracy parameter. This is the "tol" parameter passed to Arpack. Arpack's default value for that is machine precision which I guess is about 1e-15 or so. That's what I use in my other solver and it's fine. But anything better than 0.01 would help.
